src/test/resources/log4j2.properties

```diff
@@ -0,0 +1,10 @@
+# Logging settings for unit tests
+
+# The root logger with appender name
+rootLogger=DEBUG, STDOUT
```

Comment:
I'm open to criticism here in regards to scope.  If not, please mark as resolved. 🍻